src/lib/y2storage/storage_features_list.rb

```diff
@@ -38,13 +34,22 @@ class StorageFeaturesList
     def_delegators :@features, :each, :empty?, :size, :length
 
     # Constructor
+    def initialize(*features)
+      @features = [*features].flatten
```

Comment:
np: Not sure, but maybe you can achieve the same with `Array(features)`.

Reply:
No (that was also my first attempt). ;-P